### PR TITLE
feat(tokens): add z-index tokens with fallback values

### PR DESCRIPTION
**Pull request by marcosmoura**
This PR adds the z-index token values that were discussed in the [RFC: global z-index handling for v9 components](https://github.com/microsoft/fluentui/pull/31831)

Basically, this PR exposes the following:

```ts
// packages/tokens/src/global/zIndexes.ts

import type { ZIndexTokens } from '../types';

/**
 * ZIndex global defaults
 */
export const zIndexesDefaults: ZIndexTokens = {
  /**
   * Elevation 0
   * Can be used for background elements, like surfaces
   */
  zIndexBackground: '0',

  /**
   * Elevation 2
   * Can be used content that is on top of the background, like cards
   */
  zIndexContent: '1',

  /**
   * Elevation 4
   * Can be used for overlays, like the backdrop of a modal
   */
  zIndexOverlay: '1000',

  /**
   * Elevation 8
   * Can be used for popups, like modals and drawers
   */
  zIndexPopup: '2000',

  /**
   * Elevation 16
   * Can be used for messages, like snackbars and toasts
   */
  zIndexMessages: '3000',

  /**
   * Elevation 28
   * Can be used for floating elements, like dropdowns
   */
  zIndexFloating: '4000',

  /**
   * Elevation 64
   * Can be used for high priority floating elements, like tooltips
   */
  zIndexPriority: '5000',

  /**
   * Special elevation
   * Can be used for elements that need to be above everything else, like debug overlays
   */
  zIndexDebug: '6000',
};
```

```ts
// Fluent/partner code

import { tokens } from '@fluentui/theme';

const styles = {
  root: {
    zIndex: tokens.zIndexOverlay /* var(--zIndexOverlay, 1000) */,
  },
};
```

Please refer to the RFC document for more context.

---

### Related
#31831

### CODE REVIEW

File: packages/tokens/etc/tokens.api.md [hunk 1]
```diff
@@ -449,7 +449,7 @@ export const teamsHighContrastTheme: Theme;
 export const teamsLightTheme: Theme;
```
Comment (fabricteam) [inline packages/tokens/etc/tokens.api.md:449]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentui-web-components-v3 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32356&commitId=c92108ef3cebbbf3e76fe4a7bc1c7e543a78cb84&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Image.chromium.png|248| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports | Baseline (minified/GZIP) | PR    | Change     |
| :---------------- | -----------------------: | ----: | ---------: |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr> | `25.954 kB`<br />`8.595 kB` | `54.118 kB`<br />`11.917 kB` | `28.164 kB` <br />`3.322 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr> | `26.733 kB`<br />`8.872 kB` | `54.897 kB`<br />`12.221 kB` | `28.164 kB` <br />`3.349 kB` |
| <samp>react-breadcrumb</samp> <br /> <abbr title='bundle-size/index.fixture.js'>@fluentui/react-breadcrumb - package</abbr> | `113.699 kB`<br />`31.498 kB` | `141.821 kB`<br />`35.18 kB` | `28.122 kB` <br />`3.682 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr> | `101.238 kB`<br />`28.624 kB` | `129.398 kB`<br />`32.19 kB` | `28.16 kB` <br />`3.566 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr> | `94.022 kB`<br />`26.806 kB` | `122.18 kB`<br />`30.151 kB` | `28.158 kB` <br />`3.345 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr> | `69.14 kB`<br />`20.137 kB` | `69.417 kB`<br />`20.256 kB` | `277 B` <br />`119 B` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr> | `44.447 kB`<br />`14.59 kB` | `44.724 kB`<br />`14.7 kB` | `277 B` <br />`110 B` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ReactComponents.fixture.js'>react-components: entire library</abbr> | `1.097 MB`<br />`271.073 kB` | `1.098 MB`<br />`271.205 kB` | `457 B` <br />`132 B` |
| <samp>react-datepicker-compat</samp> <br /> <abbr title='bundle-size/DatePicker.fixture.js'>DatePicker Compat</abbr> | `223.64 kB`<br />`63.299 kB` | `252.028 kB`<br />`67.29 kB` | `28.388 kB` <br />`3.991 kB` |
| <samp>react-dialog</samp> <br /> <abbr title='bundle-size/Dialog.fixture.js'>Dialog (including children components)</abbr> | `99.724 kB`<br />`29.912 kB` | `127.854 kB`<br />`33.719 kB` | `28.13 kB` <br />`3.807 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr> | `28.014 kB`<br />`9.444 kB` | `56.389 kB`<br />`12.894 kB` | `28.375 kB` <br />`3.45 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/Select.fixture.js'>Select</abbr> | `27.724 kB`<br />`10.123 kB` | `56.271 kB`<br />`13.592 kB` | `28.547 kB` <br />`3.469 kB` |
| <samp>react-swatch-picker</samp> <br /> <abbr title='bundle-size/index.fixture.js'>@fluentui/react-swatch-picker - package</abbr> | `103.736 kB`<br />`30.073 kB` | `131.866 kB`<br />`34.328 kB` | `28.13 kB` <br />`4.255 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/DataGrid.fixture.js'>DataGrid</abbr> | `160.296 kB`<br />`45.518 kB` | `160.458 kB`<br />`45.536 kB` | `162 B` <br />`18 B` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/PrimitivesOnly.fixture.js'>Table (Primitives only)</abbr> | `42.481 kB`<br />`13.802 kB` | `42.643 kB`<br />`13.82 kB` | `162 B` <br />`18 B` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableAsDataGrid.fixture.js'>Table as DataGrid</abbr> | `131.15 kB`<br />`36.354 kB` | `131.312 kB`<br />`36.37 kB` | `162 B` <br />`16 B` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSelectionOnly.fixture.js'>Table (Selection only)</abbr> | `70.337 kB`<br />`19.943 kB` | `70.499 kB`<br />`19.962 kB` | `162 B` <br />`19 B` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSortOnly.fixture.js'>Table (Sort only)</abbr> | `68.98 kB`<br />`19.555 kB` | `69.142 kB`<br />`19.573 kB` | `162 B` <br />`18 B` |
| <samp>react-tags</samp> <br /> <abbr title='bundle-size/tag.fixture.js'>Tag</abbr> | `29.027 kB`<br />`9.528 kB` | `57.212 kB`<br />`12.966 kB` | `28.185 kB` <br />`3.438 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/SingleThemeToken.fixture.js'>Single theme token import</abbr> | `69 B`<br />`89 B` | `28.203 kB`<br />`3.17 kB` | `28.134 kB` <br />`3.081 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsAllThemes.fixture.js'>Teams: all themes</abbr> | `35.666 kB`<br />`7.453 kB` | `35.955 kB`<br />`7.604 kB` | `289 B` <br />`151 B` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsLightTheme.fixture.js'>Teams: Light theme</abbr> | `19.642 kB`<br />`5.549 kB` | `19.919 kB`<br />`5.695 kB` | `277 B` <br />`146 B` |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports | Size (minified/GZIP) |
| ----------------- | -------------------: |
| <samp>global-context</samp> <br /> <abbr title='bundle-size/createContext.fixture.js'>createContext</abbr> | `510 B`<br />`328 B` |
| <samp>global-context</samp> <br /> <abbr title='bundle-size/createContextSelector.fixture.js'>createContextSelector</abbr> | `537 B`<br />`339 B` |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr> | `104.529 kB`<br />`32.097 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr> | `49.303 kB`<br />`15.815 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroup.fixture.js'>AvatarGroup</abbr> | `20.106 kB`<br />`7.968 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroupItem.fixture.js'>AvatarGroupItem</abbr> | `63.447 kB`<br />`20.034 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr> | `25.719 kB`<br />`9.457 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr> | `37.104 kB`<br />`10.766 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr> | `43.518 kB`<br />`12.068 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr> | `41.916 kB`<br />`12.119 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr> | `49.93 kB`<br />`13.696 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr> | `53.037 kB`<br />`12.528 kB` |
| <samp>react-calendar-compat</samp> <br /> <abbr title='bundle-size/Calendar.fixture.js'>Calendar Compat</abbr> | `149.573 kB`<br />`39.871 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr> | `14.355 kB`<br />`5.79 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr> | `16.878 kB`<br />`6.666 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr> | `14.42 kB`<br />`5.922 kB` |
| <samp>react-checkbox</samp> <br /> <abbr title='bundle-size/Checkbox.fixture.js'>Checkbox</abbr> | `35.105 kB`<br />`12.075 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox (including child components)</abbr> | `104.398 kB`<br />`34.199 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Dropdown.fixture.js'>Dropdown (including child components)</abbr> | `104.984 kB`<br />`34.114 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `218.197 kB`<br />`63.203 kB` |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr> | `21.328 kB`<br />`7.953 kB` |
| <samp>react-field</samp> <br /> <abbr title='bundle-size/Field.fixture.js'>Field</abbr> | `23.384 kB`<br />`8.891 kB` |
| <samp>react-image</samp> <br /> <abbr title='bundle-size/Image.fixture.js'>Image</abbr> | `15.36 kB`<br />`6.236 kB` |
| <samp>react-jsx-runtime</samp> <br /> <abbr title='bundle-size/Classic.fixture.js'>Classic Pragma</abbr> | `1.057 kB`<br />`530 B` |
| <samp>react-jsx-runtime</samp> <br /> <abbr title='bundle-size/DevRuntime.fixture.js'>JSX Dev Runtime</abbr> | `3.771 kB`<br />`1.643 kB` |
| <samp>react-jsx-runtime</samp> <br /> <abbr title='bundle-size/Runtime.fixture.js'>JSX Runtime</abbr> | `4.367 kB`<br />`1.874 kB` |
| <samp>react-label</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Label</abbr> | `14.671 kB`<br />`5.99 kB` |
| <samp>react-link</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Link</abbr> | `17.192 kB`<br />`6.98 kB` |
| <samp>react-list-preview</samp> <br /> <abbr title='bundle-size/List.fixture.js'>List</abbr> | `88.626 kB`<br />`26.435 kB` |
| <samp>react-list-preview</samp> <br /> <abbr title='bundle-size/ListItem.fixture.js'>ListItem</abbr> | `112.155 kB`<br />`33.274 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr> | `151.856 kB`<br />`45.848 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu  (including selectable components)</abbr> | `154.537 kB`<br />`46.326 kB` |
| <samp>react-message-bar</samp> <br /> <abbr title='bundle-size/Default.fixture.js'>MessageBar (all components)</abbr> | `24.722 kB`<br />`9.205 kB` |
| <samp>react-motion</samp> <br /> <abbr title='bundle-size/createMotionComponent.fixture.js'>@fluentui/react-motion - createMotionComponent()</abbr> | `4.303 kB`<br />`1.899 kB` |
| <samp>react-motion</samp> <br /> <abbr title='bundle-size/createPresenceComponent.fixture.js'>@fluentui/react-motion - createPresenceComponent()</abbr> | `5.038 kB`<br />`2.229 kB` |
| <samp>react-motion</samp> <br /> <abbr title='bundle-size/PresenceGroup.fixture.js'>@fluentui/react-motion - PresenceGroup</abbr> | `1.714 kB`<br />`819 B` |
| <samp>react-overflow</samp> <br /> <abbr title='bundle-size/hooks.fixture.js'>hooks only</abbr> | `12.81 kB`<br />`4.821 kB` |
| <samp>react-persona</samp> <br /> <abbr title='bundle-size/Persona.fixture.js'>Persona</abbr> | `56.194 kB`<br />`17.695 kB` |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr> | `128.329 kB`<br />`40.143 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr> | `14.563 kB`<br />`5.118 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr> | `8.39 kB`<br />`2.64 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePositioning</abbr> | `27.057 kB`<br />`9.698 kB` |
| <samp>react-progress</samp> <br /> <abbr title='bundle-size/ProgressBar.fixture.js'>ProgressBar</abbr> | `17.084 kB`<br />`6.891 kB` |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr> | `24.623 kB`<br />`8.893 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/Radio.fixture.js'>Radio</abbr> | `32.672 kB`<br />`10.343 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/RadioGroup.fixture.js'>RadioGroup</abbr> | `15.762 kB`<br />`6.423 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/Slider.fixture.js'>Slider</abbr> | `37.169 kB`<br />`12.538 kB` |
| <samp>react-spinbutton</samp> <br /> <abbr title='bundle-size/SpinButton.fixture.js'>SpinButton</abbr> | `36.102 kB`<br />`11.827 kB` |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr> | `25.245 kB`<br />`8.539 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr> | `35.319 kB`<br />`11.314 kB` |
| <samp>react-tag-picker</samp> <br /> <abbr title='bundle-size/Package.fixture.js'>@fluentui/react-tag-picker - package</abbr> | `182.31 kB`<br />`54.807 kB` |
| <samp>react-tags</samp> <br /> <abbr title='bundle-size/InteractionTag.fixture.js'>InteractionTag</abbr> | `15.199 kB`<br />`6.157 kB` |
| <samp>react-tags</samp> <br /> <abbr title='bundle-size/TagGroup.fixture.js'>TagGroup</abbr> | `82.197 kB`<br />`24.362 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Default.fixture.js'>Text - Default</abbr> | `17.061 kB`<br />`6.723 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Wrappers.fixture.js'>Text - Wrappers</abbr> | `20.242 kB`<br />`7.048 kB` |
| <samp>react-textarea</samp> <br /> <abbr title='bundle-size/Textarea.fixture.js'>Textarea</abbr> | `26.572 kB`<br />`9.755 kB` |
| <samp>react-timepicker-compat</samp> <br /> <abbr title='bundle-size/TimePicker.fixture.js'>TimePicker</abbr> | `107.387 kB`<br />`35.758 kB` |
| <samp>react-toast</samp> <br /> <abbr title='bundle-size/Toast.fixture.js'>Toast (including Toaster)</abbr> | `97.816 kB`<br />`29.438 kB` |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr> | `55.517 kB`<br />`19.515 kB` |
| <samp>react-utilities</samp> <br /> <abbr title='bundle-size/SSRProvider.fixture.js'>SSRProvider</abbr> | `180 B`<br />`160 B` |
</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/843386fc13e12d9efca29eb1e26a2654239167b6'>843386fc13e12d9efca29eb1e26a2654239167b6</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32356/perf-test-react-components/Avatar-mount_base.html">640</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32356/perf-test-react-components/Avatar-mount.html">638</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32356/perf-test-react-components/Button-mount_base.html">308</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32356/perf-test-react-components/Button-mount.html">305</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32356/perf-test-react-components/Field-mount_base.html">1132</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32356/perf-test-react-components/Field-mount.html">1134</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32356/perf-test-react-components/FluentProvider-mount_base.html">708</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32356/perf-test-react-components/FluentProvider-mount.html">724</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32356/perf-test-react-components/FluentProviderWithTheme-mount_base.html">84</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32356/perf-test-react-components/FluentProviderWithTheme-mount.html">108</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32356/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">37</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32356/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">41</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32356/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">77</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32356/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">89</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32356/perf-test-react-components/MakeStyles-mount_base.html">870</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32356/perf-test-react-components/MakeStyles-mount.html">854</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32356/perf-test-react-components/Persona-mount_base.html">1752</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32356/perf-test-react-components/Persona-mount.html">1713</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32356/perf-test-react-components/SpinButton-mount_base.html">1380</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32356/perf-test-react-components/SpinButton-mount.html">1368</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32356/perf-test-react-components/SwatchPicker-mount_base.html">1641</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32356/perf-test-react-components/SwatchPicker-mount.html">1665</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: change/@fluentui-tokens-6453c666-41f3-4fb9-bba7-ca79fa23508d.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-tokens-6453c666-41f3-4fb9-bba7-ca79fa23508d.json:1]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32356&commitId=c92108ef3cebbbf3e76fe4a7bc1c7e543a78cb84&env=prod&clientType=FLUENTUI)


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|974| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

File: packages/tokens/src/tokens.ts [hunk 3]
```diff
@@ -539,4 +540,8 @@ export const tokens: Record<keyof Theme, string> = {
   curveEasyEaseMax: 'var(--curveEasyEaseMax)',
   curveEasyEase: 'var(--curveEasyEase)',
   curveLinear: 'var(--curveLinear)',
+
+  // ZIndexes
+  // This is a special case where the tokens contain default values
+  ...zIndexVarsWithDefaults,
```
Comment (layershifter) [inline packages/tokens/src/tokens.ts:546]: Let's just inline it there and avoid an additional file `./global/zIndexes`, i.e.:


```suggestion
  /**
   * Elevation 0
   * Can be used for background elements, like surfaces
   */
  zIndexBackground: 'var(--zIndexBackground, 0)',
  
  // ....
```

File: packages/tokens/src/tokens.test.ts [hunk 4]
```diff
@@ -4,7 +4,18 @@ import { tokens } from './tokens';
...
+
+      if (hasFallback) {
+        const [, fallback] = value.replace(/ |\)/g, '').split(',');
+
+        // In case of a fallback, we expect the value to be in the format `var(--token, fallback)`.
+        expect(value).toBe(`var(--${token}, ${fallback})`);
+      } else {
+        // In case of no fallback, we expect the value to be in the format `var(--token)`.
+        expect(value).toBe(`var(--${token})`);
+      }
```
Comment (layershifter) [inline packages/tokens/src/tokens.test.ts:18]:
```suggestion
      const tokenRegex = new RegExp(`var\\(--${token}(, .+)?\\)`)
      const tokenValue = tokens[token];
      
      expect(tokenValue).toMatch(tokenRegex);
```

IMO a regex will be nicer there ;)

File: packages/tokens/src/themeToTokensObject.test.ts [hunk 5]
```diff
@@ -1,11 +1,21 @@
...
+    expect(tokensKeys.includes(token)).toBe(true);
+  });
+}
 
 describe('themeToTokensObject', () => {
   it('passing any of our default themes to the function generates the default tokens object', () => {
     const generatedTokens = themeToTokensObject(webLightTheme);
-    expect(generatedTokens).toEqual(tokens);
+
+    assertKeys(generatedTokens);
```
Comment (layershifter) [inline packages/tokens/src/themeToTokensObject.test.ts:18]:
```suggestion
  expect(tokens).toMatchObject(generatedTokens);
```

there is a built-in assertion for this ;)

File: change/@fluentui-tokens-6453c666-41f3-4fb9-bba7-ca79fa23508d.json (hunk 2 above)
Comment (fabricteam) [inline change/@fluentui-tokens-6453c666-41f3-4fb9-bba7-ca79fa23508d.json:1]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentui-web-components-v3 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32356&commitId=5ef1e209db6a668482cdeb5e79c42c65d3531912&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Image.chromium.png|248| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

File: packages/tokens/src/themeToTokensObject.test.ts [hunk 6]
```diff
@@ -17,6 +27,7 @@ describe('themeToTokensObject', () => {
       customColor3: 'var(--customColor3)',
     };
     const generatedTokens = themeToTokensObject(customTheme);
-    expect(generatedTokens).toEqual(expectedTokens);
+
+    assertKeys(generatedTokens, expectedTokens);
```
[the remark this answers is not in the document]
Comment (marcosmoura) [reply packages/tokens/src/themeToTokensObject.test.ts:31]: Same here.

File: packages/tokens/src/themeToTokensObject.test.ts (hunk 5 above)
Comment (marcosmoura) [reply packages/tokens/src/themeToTokensObject.test.ts:19]: I added another check to test the object format (rather than the full value)

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Pull request merged**